### DISCUSSION TITLE
Formalising cohomology theories

### DISCUSSION THREAD

**Discussion by giscus** [leanprover-community/blog#70] [Announcements]
# Formalising cohomology theories

Kevin Buzzard rounds up the BIRS conference on formalising cohomology theories.

Formalising cohomology theories
At the end of May a group of formalisation enthusiasts, many of them Lean experts, met 

https://leanprover-community.github.io/blog/posts/banff-cohomology/

**Comment by kim-em** [comment]
The final report on the workshop is now available at https://www.birs.ca/files//workshops/Juniper/23w5124/report23w5124.pdf, with comments from many participants!
